Misc/NEWS.d/next/Tests/2019-05-04-21-25-19.bpo-36782.h3oPIb.rst

```diff
@@ -0,0 +1,9 @@
+Created tests for these C API functions 
```

Comment:
Just Add tests for C API functions. you don't need to specify the names of all the tested functions. One or two lines for the description is fine.